### DISCUSSION TITLE
Make themes/ folder from doks

### DISCUSSION THREAD

**Discussion by patelniket** [thuliteio/doks#867] [Q&A]
Hello, apologies I am a novice with hugo and npm dependencies management and I know this question has come up before, but is there any possible way that I can have doks inside a hugo themes/ folder?. If one was to do this manually then what would be the steps for that?. I understand that managing updates may be a manual effort with this approach.

The reason I ask is because we use a built-in hugo documentation site builder which supports users bringing their own custom theme but excepts it to be in a themes/ folder. And I would really like to use doks with that structure because the theme is simple and solid for building documentation sites.

**Comment by h-enk** [comment]
I'm currently (one of my main priorities) looking into restructuring Doks + the possibilities of aligning with the default Hugo setup. When I've got things clear, I'll get back to you.

**Reply by patelniket** [reply]
Thank you very much @h-enk!. That would be wonderful. I will be looking forward to this update!

In the meantime though is there any short-term solution that I can do for this by moving around some folders into themes maybe?. If am okay if it takes some manual work but some pointers for this would be helpful!.
